src/mixin/java/io/github/brassmc/brassloader/mixin/MinecraftServerMixin.java

```diff
@@ -0,0 +1,20 @@
+package io.github.brassmc.brassloader.mixin;
+
+import net.minecraft.obfuscate.DontObfuscate;
+import net.minecraft.server.MinecraftServer;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+
+@Mixin(MinecraftServer.class)
+public class MinecraftServerMixin {
+
+    /**
+     * @author BrassMc
+     * @reason change server mod name to brass
+     */
+    @DontObfuscate
+    @Overwrite(remap = false)
```

Comment:
The remap here seems pointless as we've decided not to have obfuscated mappings at runtime. But I guess it's fine for future-proofing.

```suggestion
    @Overwrite
```